tests/src/core/Polyfill.test.ts

```
@@ -35,4 +35,39 @@ describe("Polyfill", () => {
       expect(resultArray[2][1]).to.equal(originResultArray[2][1]).to.equal("/f3.glsl");
     });
   });
+
+  it("AudioContext polyfill", async () => {
+    delete window.AudioContext;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)